test/evergreen.yml

```diff
@@ -125,7 +125,7 @@ functions:
         # Fetch the gperftools library.
         if [[ "${posix_configure_flags|}" =~ (tcmalloc|TCMALLOC) ]]; then
           git clone git@github.com:wiredtiger/automation-scripts.git
-          . automation-scripts/evergreen/find_gperftools.sh ${s3_access_key} ${s3_secret_key} ${build_variant} ${is_cmake_build|false}
+          . automation-scripts/evergreen/find_gperftools.sh ${s3_access_key} ${s3_secret_key} ${build_variant} true
```

Comment:
Can we instead create the variable `is_cmake_build` and set it to true and leave `${is_cmake_build}`?
having a variable `is_cmake_build` will be easier to understand than a simple `true`.

Reply:
Done: 448fa179ea163847ea7d30467e23dce9385076f7